### PR TITLE
Minion Spawner

### PR DESCRIPTION
**Pull request by NamespaceV**
image

### CODE REVIEW

File: Assets/Scripts/Runtime/Interactable/Spawner.cs [hunk 1]
```diff
@@ -0,0 +1,43 @@
+using Assets.Scripts.Runtime.Character;
+using UnityEditor;
+using UnityEngine;
+
+public class Spawner : MonoBehaviour
+{
+    [SerializeField] public Minion minionPrefab;
```
Comment (klawir) [inline Assets/Scripts/Runtime/Interactable/Spawner.cs]: why  [SerializeField] public?

Comment (klawir) [reply Assets/Scripts/Runtime/Interactable/Spawner.cs]: private

File: Assets/Scripts/Runtime/Interactable/Spawner.cs [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    [SerializeField] public MeshRenderer litMesh;
+    [SerializeField] public Material notLitMaterial;
+    [SerializeField] public Material litMaterial;
+
+    static Mesh capsuleMesh;
```
Comment (klawir) [inline Assets/Scripts/Runtime/Interactable/Spawner.cs]: private by default :)

File: Assets/Scripts/Runtime/Character/Hero.cs [hunk 3]
```diff
@@ -58,6 +99,21 @@ private void initializeForMinions()
...
+            _minionsThatAreNotExecutingAnOrder.Add(m);
+            
+            localThirdPersonController.OnJumpEnd += m.FollowHero;
+            localThirdPersonController.OnMove += m.FollowHero;
+            m.OnFishedOrder += reactivatePassiveFollowHero;
+        }
+
+        public bool canGetAnotherMinion(){
+            Debug.Log($"Minions count {minions.Count}");
+            return minions.Count < 10;
```
Comment (klawir) [inline Assets/Scripts/Runtime/Character/Hero.cs]: make a const :D

File: Assets/Scripts/Runtime/Interactable/Spawner.cs (hunk 1 above)
Comment (NamespaceV) [reply Assets/Scripts/Runtime/Interactable/Spawner.cs]: fixed

File: Assets/Scripts/Runtime/Interactable/Spawner.cs (hunk 2 above)
Comment (NamespaceV) [reply Assets/Scripts/Runtime/Interactable/Spawner.cs]: fixed

File: Assets/Scripts/Runtime/Character/Hero.cs (hunk 3 above)
Comment (NamespaceV) [reply Assets/Scripts/Runtime/Character/Hero.cs]: fixed

**Review by klawir — APPROVED** [review]

**Review by klawir — APPROVED** [review]

**Pull request merged**